src/CardGrid.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FunctionComponent } from 'react';
+import Card from './Card';
+
+interface Pokemon {
```

Comment:
Create a `src/types/index.ts` file for storing type defs we'll use in multiple places.